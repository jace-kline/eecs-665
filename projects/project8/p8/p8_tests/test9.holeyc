int main() {
    int a;
    int b;
    int c;
    a = 8;
    b = 6;
    a = b;
    return a;
}