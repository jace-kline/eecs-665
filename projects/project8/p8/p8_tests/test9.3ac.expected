[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN main LOCALS]
a (local)
b (local)
c (local)
[END main LOCALS]
main:       enter main
            [a] := 8
            [b] := 6
            [a] := 6
            setret 6
lbl_0:      leave main
